setup.py

```diff
@@ -167,7 +167,8 @@
     install_requires=[
         'click-default-group',
         'click',
-        'natsort'
+        'natsort',
+        'deepdiff==5.2.3'
```

Comment:
>deepdiff [](start = 9, length = 8)

This new dependency is tests_require. If you put it here, it will installed in sonic image.

Reply:
Fixed, thanks